Makefile

```diff
@@ -15,7 +15,7 @@ RELATIVE=$(MDS:.md=.rmd)
 
 SLIDES=$(MDS:.md=-reveal.html)
 
-EXES=$(shell find build -type f -perm +111 -name *.x)
+EXES=$(shell find build -type f -perm /111 -name *.x)
```

Comment:
Is this change compatible with bsd find on macs?

Reply:
Good catch. The BSD and (newer) GNU `find` options for specifying permissions are not compatible with each other. Replaced it with the POSIX equivalent.